client-scala/src/main/scala/com/influxdb/client/scala/internal/WriteScalaApiImpl.scala

```diff
@@ -101,13 +99,12 @@ class WriteScalaApiImpl(@Nonnull service: WriteService, @Nonnull options: Influx
    * @param org    Specifies the destination organization for writes.
    *               The [[com.influxdb.client.InfluxDBClientOptions#getOrg]] will be used as the destination `organization`
    *               if the `org` is not specified.
-   * @return the sink that accept the Data points. The `point` is considered as one batch unit.
+   * @return the sink that accept the Data points.
    */
   override def writePoint(bucket: Option[String], org: Option[String]): Sink[Point, Future[Done]] = {
     Flow[Point]
       .map(point => (point.getPrecision, Seq(new AbstractWriteClient.BatchWriteDataPoint(point, options))))
-      .map(batch => writeHttp(Some(batch._1), bucket, org, batch._2))
-      .toMat(Sink.head)(Keep.right)
+      .toMat(Sink.foreach(batch =>writeHttp(Some(batch._1), bucket, org, batch._2)))(Keep.right)
```

Comment:
Just typo:

```suggestion
      .toMat(Sink.foreach(batch => writeHttp(Some(batch._1), bucket, org, batch._2)))(Keep.right)
```

Reply:
I completely missed that, good catch. Fixed